src/amo/css/AddonReviewList.scss

```diff
@@ -40,6 +41,13 @@
 
   .AddonReviewList-header-icon {
     margin: 0 20px;
+    width: 54px;
+    height: 54px;
+  }
+
+  .AddonReviewList-header-icon img {
+    width: 54px;
```

Comment:
Heads up!
This results in stretching of image to 54x54 which is actually 32x32.